backend/app/services/common.js

```diff
@@ -624,6 +637,36 @@ var copyAssessmentAtSurveyService = function (assessmentId, prevAssessmentId, jw
 
 exports.copyAssessmentAtSurveyService = copyAssessmentAtSurveyService;
 
+var getAssessmentStatusAtSurveyService = function (assessmentId, jwt) {
+    const path = 'assessment-answers/';
+    const path2 = '/status';
+
+    const requestOptions = {
+        url: config.surveyService + path + assessmentId + path2,
+        method: 'GET',
+        headers: {
+            'authorization': jwt,
+            'origin': config.domain
+        },
+        resolveWithFullResponse: true,
+    };
+
+    return request(requestOptions)
+        .then((res) => {
+            if (res.statusCode > 299 || res.statusCode < 200) {
+                const httpErr = new HttpError(res.statusCode, res.statusMessage);
+                return Promise.reject(httpErr);
+            }
+            return res;
+        })
+        .catch((err) => {
+            const httpErr = new HttpError(500, `Unable to use survey service: ${err.message}`);
+            return Promise.reject(httpErr);
+        });
```

Comment:
It's probably time we make this block of code its own function, we use is in a lot of different places now. But for a separate ticket
https://jira.amida-tech.com/browse/INBA-739